pkg/statistics/mock_test.go

```diff
@@ -5,3 +5,12 @@ package statistics
 //go:generate mockgen --build_flags=--mod=mod -destination=./mock/keycloak_client.go -package=mock -mock_names=KeycloakClient=KeycloakClient github.com/cloudtrust/common-service/v2/security KeycloakClient
 //go:generate mockgen --build_flags=--mod=mod -destination=./mock/authentication_db_reader.go -package=mock -mock_names=AuthorizationDBReader=AuthorizationDBReader github.com/cloudtrust/common-service/v2/security AuthorizationDBReader
 //go:generate mockgen --build_flags=--mod=mod -destination=./mock/accreditationsclient.go -package=mock -mock_names=AccreditationsServiceClient=AccreditationsServiceClient github.com/cloudtrust/keycloak-bridge/internal/keycloakb/accreditationsclient AccreditationsServiceClient
+//go:generate mockgen --build_flags=--mod=mod -destination=./mock/idnowclient.go -package=mock -mock_names=IdnowServiceClient=IdnowServiceClient github.com/cloudtrust/keycloak-bridge/internal/keycloakb/idnowclient IdnowServiceClient
+
+func ptr(value string) *string {
+	return &value
+}
+
+func intPTR(value int) *int {
```

Comment:
We usually use intPtr(...)

Reply:
Yeah, forgot that. I do not like it as it would be more standard in Go to call it intPTR. But let be consistent